ci/vm.sh

```diff
@@ -144,6 +141,8 @@ gen_metadata() {
 		  "wipe_disks": true
 		}
 	EOF
+	mkdir -p 2009-04-04/meta-data
```

Comment:
Please document this magic value (2009-04-04) or set it to a variable with a describing string. I imagine that this value has to be identical across a couple of places here.

Reply:
done